understanding/20/change-on-request.html

```diff
@@ -56,8 +56,8 @@ <h2>Benefits of Change on Request</h2>
          
          <li>
             
-            <p>Individuals who are unable to detect changes of context or may not realize that the
-               context has changed are less likely to become disoriented while navigating a site.
+            <p>Individuals who are unable to detect changes of context, or may not realize that the
+               context has changed if they did not initiate the change are less likely to become disoriented while navigating a site.
```

Comment:
```suggestion
            <p>Individuals who are unable to detect changes of context, or may not realize that the
               context has changed if they did not initiate the change, are less likely to become disoriented while navigating a site.
```

Reply:
I *think* the above suggestion would gets to what @jake-abma was saying? (though admittedly, I wasn't 100% sure about what he was getting at, myself)

Reply:
I think just adding a second 'who' clarifies
```suggestion
            <p>Individuals who are unable to detect changes of context or who may not realize that the context has changed are less likely to become disoriented while navigating a site.
```

Reply:
but both those sentences say the same thing, unless i'm missing something? "people who are unable to detect change, or people who are unable to detect change if they didn't initiate it" ...